cmd/server/main.go

```diff
@@ -1,69 +1,32 @@
 package main
 
 import (
-	"fmt"
 	"net/http"
-	"os"
 
 	"log"
 
-	"github.com/golang-migrate/migrate/v4"
-	_ "github.com/golang-migrate/migrate/v4/database/postgres"
-	_ "github.com/golang-migrate/migrate/v4/source/file"
-	_ "github.com/jackc/pgx/v4/stdlib"
-	"github.com/jmoiron/sqlx"
+	"github.com/broadinstitute/sherlock/internal/db"
+	"github.com/broadinstitute/sherlock/internal/sherlock"
+	"github.com/broadinstitute/sherlock/internal/tools"
 )
 
-const changelogLocation = "file://db/migrations"
-
 func main() {
-	db, err := sqlx.Connect("pgx", os.Getenv("POSTGRESQL_URL"))
-	if err != nil {
-		log.Fatalf("Unable to connect to database: %v\n", err)
-		os.Exit(1)
-	}
-	defer db.Close()
-
-	log.Println("Successfully connected to database")
 
 	// just logging error rather than failing here as
 	// a no changes migration is treated as an error..
-	if err := applyMigrations(); err != nil {
+	if err := db.ApplyMigrations("db/migrations", sherlock.Config); err != nil {
```

Comment:
I think eventually we don't want to run the migrations in app at all and use something like Jack's liquibase migration job.

I only really see this being enabled in local  development and integration test workflows. I can make it fail if you feel strongly though. I mainly had it as a pass through because the failures were annoying me when I was trying to fix things :)